Error in 'Seed the database' Code?

Hi all,

I was getting an error:
c:\Users\UserName\Projects\MvcMovie\Program.cs(24,63): error CS0246: The type or namespace name 'MvcMovieContext' could not be found (are you missing a using directive or an assembly reference?) [C:\Users\UserName\Projects\MvcMovie\MvcMovie.csproj]

Until I changed the below snippet:

using Microsoft.AspNetCore;
using Microsoft.AspNetCore.Hosting;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Logging;
using System;
using Microsoft.EntityFrameworkCore;
using MvcMovie.Models;
using MvcMovie;

to the following:

using Microsoft.AspNetCore;
using Microsoft.AspNetCore.Hosting;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Logging;
using System;
using Microsoft.EntityFrameworkCore;
using MvcMovie.Models;
using MvcMovie.Data;

Does this need to be updated?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a0a4022c-7ef1-3c30-4f9d-cf521a57a44a
* Version Independent ID: a0b5e15c-d747-8710-95fb-a55c09861a1e
* Content: [Work with SQL in an ASP.NET Core MVC app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/working-with-sql?view=aspnetcore-3.0&tabs=visual-studio-code#seed-the-database)
* Content Source: [aspnetcore/tutorials/first-mvc-app/working-with-sql.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/first-mvc-app/working-with-sql.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

--- a/aspnetcore/tutorials/first-mvc-app/working-with-sql.md
+++ b/aspnetcore/tutorials/first-mvc-app/working-with-sql.md
@@ -87,7 +87,7 @@ if (context.Movie.Any())
 
 Replace the contents of *Program.cs* with the following code:
 
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie22/Program.cs)]
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Program.cs)]
 
 Test the app
 

